aggregator/utils/utils.py

```diff
@@ -45,14 +47,17 @@ async def parse_version(semver):
 async def http_get_service_urls(registry):
     """Query an external registry for known service urls of desired type."""
     LOG.debug("Query external registry for given service type.")
+    LOG.info("registry inside http_get_service_urls:"+str(registry))

```

Comment:
use either f-string or % for this kind of change (% preferred for performance, however the current code style follows f-string
```suggestion
    LOG.info(f"registry inside http_get_service_urls: {registry}")

```